lib/transaction.js

```diff
@@ -44,6 +43,10 @@ function mixinTransaction(PostgreSQL) {
   PostgreSQL.prototype.commit = function(connection, cb) {
     debug('Commit a transaction');
     var self = this;
+    if (connection instanceof Transaction) {
```

Comment:
I've tested our changes in `loopback-connector` and it works without any changes in this repo. 
Any reason we need to make this change?